templates/switchers.js

```diff
@@ -18,8 +18,8 @@
     '(?:dev)',
     '(?:release/\\d.\\d[\\x\\d\\.]*)'];
 
-  var all_versions = $VERSIONS;
-  var all_languages = $LANGUAGES;
+  var all_versions = {"3.13": "dev (3.13)", "3.12": "pre (3.12)", "3.11": "3.11", "3.10": "3.10", "3.9": "3.9", "3.8": "3.8", "3.7": "3.7", "3.6": "3.6", "3.5": "3.5", "2.7": "2.7"};
```

Comment:
Have you hardcoded those for test purposes and forgot to not commit it?

Reply:
Oops sorry, that's right, let me revert that...